enterprise-suite/es-monitor-api/prometheus.yml

```diff
@@ -207,7 +210,7 @@ scrape_configs:
       - action: labelmap
         regex: __meta_kubernetes_service_label_(.+)
       - source_labels: [__meta_kubernetes_namespace]
-        target_label: kubernetes_namespace
+        target_label: namespace
```

Comment:
Worth keeping both so we don't break the frontend?

Reply:
the one they're using is retained, hoping we can pick a label and use it consistently tho